tests/test_lock.py

```diff
@@ -262,10 +293,37 @@ def test_lock_load(multiprocessing, num_workers, timeout, expected):
 
     assert len(result) == expected
 
-    # Windows removes the lock files
-    if not sys.platform == 'win32':
-        assert os.path.exists(DB_LOCK_PATH)
-        assert os.path.exists(DB_FLAVOR_LOCK_PATH)
+    assert not os.path.exists(DB_LOCK_PATH)
+    assert not os.path.exists(DB_FLAVOR_LOCK_PATH)
+
+
+@pytest.mark.parametrize(
+    'fixture_set_repositories',
+    ['crash-file-system'],
+    indirect=True,
+)
+def test_lock_load_crash(fixture_set_repositories):
+
+    repositories_tmp = audb.config.REPOSITORIES
+
+    audb.config.REPOSITORIES = [
+        audb.Repository(
+            name=pytest.REPOSITORY_NAME,
+            host=pytest.FILE_SYSTEM_HOST,
+            backend='crash-file-system',
+        ),
+    ]
+
+    assert not os.path.exists(DB_LOCK_PATH)
+    assert not os.path.exists(DB_FLAVOR_LOCK_PATH)
+
+    with pytest.raises(RuntimeError):
+        load_db(-1)
+
+    assert not os.path.exists(DB_LOCK_PATH)
+    assert not os.path.exists(DB_FLAVOR_LOCK_PATH)
+
+    audb.config.REPOSITORIES = repositories_tmp
```

Comment:
Is this needed, we don't do it in the other tests?

Can it be added to `fixture_set_repositories`?

Reply:
Same here, the code was a left-over and was removed now.